src/js/components/sharedComponents/DatePicker.jsx

```diff
@@ -229,6 +229,10 @@ export default class DatePicker extends React.Component {
             });
         }
 
+        else if (!this.props.value) {
+            disabledDays.length = 0;
```

Comment:
Why are you changing the length of the array instead of the array itself?

Reply:
Since disabled days is a const I can either update the length property to 0 which will reset it to an empty array or splice it onto itself at position 0 or iteratively pop off each element. I thought this was the most straightforward way to do it.